src/technology-consulting/5-communication.md

```diff
@@ -9,23 +9,52 @@
 
 The following the communication tools that should be present on every project. The specifics on your project should be documented in the [learn-technology-consulting/transitioning-to-delivery#app-summary-document App Summary Document]
 
-- <input type="checkbox"/> Wiki - _GitHub Wiki_
-- <input type="checkbox"/> Screen Sharing - _Google Meet_ or _Slack Calls_
-- <input type="checkbox"/> Video Conference - _Google Meet_ or _Slack Calls_
-- <input type="checkbox"/> Chat - _Slack_
-- <input type="checkbox"/> Calendars - _Shared Google Calendars_ and a project/company calendar.
-- <input type="checkbox"/> Meetings - When meetings are held.
-
+- Wiki - _GitHub Wiki_
+- Screen Sharing - _Google Meet_ or _Slack Calls_
+- Video Conference - _Google Meet_ or _Slack Calls_
+- Chat - _Slack_
+- Calendars - _Shared Google Calendars_ and a project/company calendar.
```

Comment:
Calendars can likely be removed, as they are discussed in Transitioning to Delivery